introduction.md

```diff
@@ -0,0 +1 @@
+Hello, my name is richardkoszegi and I like CodeSchool. Every course, that I took, is very good.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.